src/codeSigning/SimplifiedSigning.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { generateKeyPair, createSign, createVerify } from 'crypto';
+import { createReadStream } from 'fs';
+import { S3 } from 'aws-sdk';
+import { putObject } from '../codeSigning/upload';
+
+const CRYPTO_LEVEL = 'RSA-SHA256';
+const BUCKET = 'dfc-data-production';
+export const BASE_URL = 'https://developer.salesforce.com';
+export const SECURITY_PATH = 'media/salesforce-cli/security';
+
+interface SigningRequest {
+  targetFileToSign: string; // path to the file on local FS
```

Comment:
Can you make these jsdoc instead? Then it will appear in autocomplete and hovers.

```
interface SigningRequest {
  /** path to the file on local FS */
  targetFileToSign: string;
  ...
```